src/components/CartProductCard.jsx

```diff
@@ -73,6 +74,7 @@ const CartProductCard = ({ product }) => {
               type: 'Remove from Cart',
```

Comment:
I am just nitpicking here, but the convention would be "REMOVE_FROM_CART". Maybe from the next project you can implement it. 😅

Reply:
Ohh will sure keep that in mind